lib/deadpool-cyclone/src/lib.rs

```diff
@@ -155,4 +158,55 @@ mod tests {
 
         instance.terminate().await.expect("failed to terminate");
     }
+
+    #[tokio::test]
+    async fn pow() {
+        let mut config_file = veritech_server::ConfigFile::default_local_uds();
+        veritech_server::detect_and_configure_development(&mut config_file)
+            .expect("failed to determine test configuration");
+
+        let spec = LocalUdsInstance::spec()
+            .try_cyclone_cmd_path(config_file.cyclone.cyclone_cmd_path())
+            .expect("failed to find cyclone program")
+            .cyclone_decryption_key_path(config_file.cyclone.cyclone_decryption_key_path())
+            .try_lang_server_cmd_path(config_file.cyclone.lang_server_cmd_path())
+            .expect("failed to find lang server program")
+            .limit_requests(2)
+            .runtime_strategy(LocalUdsRuntimeStrategy::LocalDocker)
+            .socket_strategy(LocalUdsSocketStrategy::Random)
+            .ping()
```

Comment:
What does `ping()` do? Just curious.

Reply:
It enables the `/ping` route for Cyclone so we can poke it and make sure it's still up.